### ISSUE TITLE
RecursionError on deletion

### ISSUE THREAD

**Issue by ericswpark** [jkeifer/drf-chunked-upload#14]
When deleting a ChunkedUpload object, the deletion fails because the deletion function continuously calls itself.

It only occurs when calling the `.delete()` function on a ChunkedUpload object (i.e. `chunked_upload.delete()`) or if a single ChunkedUpload object is deleted in the admin panel. For some reason it doesn't occur when bulk-deleting (i.e. checking each item in the ChunkedUpload table and delete using bulk actions).

https://github.com/jkeifer/drf-chunked-upload/blob/b6fcf85c39225617ae014d44f83e17470873cb34/drf_chunked_upload/models.py#L88

**Comment by ericswpark** [comment]
@jkeifer found the issue, it's calling `super(ChunkedUpload, self).delete(*args, **kwargs)` within the `AbstractChunkedUpload` model, but it's not shown in the linked printout because it's part of version 0.5.0a1. 

Once #13 is up as a stable release/source I can try and fix this.

**Comment by jkeifer** [comment]
Argh, I was half working on some changes on my new CI branch where I was adding testing when I pivoted to getting the migrations in place and setting up the new build stuff on that branch. I probably shouldn't have mixed that all together, but it seemed the easiest way forward. This bug is from that work.

But anyway, I just fixed that super call and built 0.5.0a2 for you. Let me know if you still have issues.
